local-tests/tests/testRelayer.ts

```diff
@@ -17,8 +16,10 @@ export const testRelayer = async (devEnv: TinnyEnvironment) => {
 
   // -- test fetch pkps
   const litRelay = new LitRelay({
-    // devEnv.network is LIT_TESTNET, which is an enum similar to LitNetwork but with less networks
-    relayUrl: LitRelay.getRelayUrl(devEnv.network as unknown as LitNetwork),
+    // devEnv.network is LIT_TESTNET, which is an enum similar to LIT_NETWORK but with less networks
+    relayUrl: LitRelay.getRelayUrl(
+      devEnv.network as unknown as LIT_NETWORK_VALUES
```

Comment:
Do we still need the casting here?

Reply:
`LIT_TESTNET` includes `localchain` but `LIT_NETWORK` does not, it [has `custom` and `localhost`](https://github.com/LIT-Protocol/js-sdk/pull/579/files#diff-f26476a636a35901110ac360f37c1f63adb607f40eb85c6c2f27cc54da0f99dfR698-R699)

Maybe we can unify?

Reply:
We do have this ticket assigned 20 days ago, if you have the bandwidth would you mind taking it since you're already working on this? https://linear.app/litprotocol/issue/LIT-3690/[v7]-refactor-lit-networks-and-remove-localhost-and-internaldev-from

Reply:
Grabbed it @MaximusHaximus

Reply:
Included changes for it in commit [678ad26](https://github.com/LIT-Protocol/js-sdk/pull/579/commits/678ad26b034ed13b43adefcd465ae1dad37b2f18)